section6/resources/item.py

```diff
@@ -52,12 +53,13 @@ def put(self, name):
         if item:
             item.price = data['price']
         else:
-            item = ItemModel(name, data['price'])
+            item = ItemModel(name, **data)
```

Comment:
We will create a new item in the `else` statement, and since the `store_id` is a required field in the `reqparser`, I suppose it should be provided to ItemModel.

Reply:
That makes sense, can you change the `post()` method too so it's the same?

Reply:
It's updated now.